source/code/plugins/filter_syslog.rb

```diff
@@ -25,12 +25,32 @@ def start
     def shutdown
       super
     end
+
+    # For perf debugging
+    def check_eps()
+      if @eps_counter == nil
+        @eps_counter = 0
```

Comment:
shouild this be logged only when debug flag is enabled?

Reply:
Not really, I only use this when I need to do EPS/benchmarking, if we add an if/else it may impact EPS little bit.